Assets/MixedRealityToolkit/Utilities/Editor/MixedRealityPreferences.cs

```diff
@@ -64,6 +64,33 @@ public static bool IgnoreSettingsPrompt
 
         #endregion Ignore startup settings prompt
 
+        #region Auto-Enable UWP Capabilities
+
+        private static readonly GUIContent AutoEnableCapabilitiesContent = new GUIContent("Auto-Enable UWP Capabilities", "When this setting is enabled, MRTK services requiring particular UWP capabilities will be auto-enabled in Publishing Settings.\n\nOnly valid for UWP Build Target projects.\n\nUWP Capabilities can be viewed under Player Settings > Publishing Settings.");
+        private const string AUTO_ENABLE_CAPABILITIES_KEY = "_MixedRealityToolkit_Editor_AutoEnableUWPCapabilities";
+        private static bool autoEnabledCapabilitiesPrefLoaded;
+        private static bool autoEnabledCapabiltiiesSettingsPrompt;
+
+        /// <summary>
+        /// Should the settings prompt show on startup?
+        /// </summary>
+        public static bool AutoEnableUWPCapabilities
+        {
+            get
+            {
+                if (!autoEnabledCapabilitiesPrefLoaded)
+                {
+                    autoEnabledCapabiltiiesSettingsPrompt = EditorPrefs.GetBool(IGNORE_KEY, true);
+                    autoEnabledCapabilitiesPrefLoaded = true;
+                }
+
+                return autoEnabledCapabiltiiesSettingsPrompt;
+            }
+            set => EditorPrefs.SetBool(IGNORE_KEY, autoEnabledCapabiltiiesSettingsPrompt = value);
```

Comment:
Not a blocker for this PR, but I have some proposals going forward to move away from `EditorPrefs` wherever possible / it makes sense, and move towards ScriptableObject-based settings.
This way (going back to "wherever it makes sense"), settings can be checked into source control and aren't local to a user. Something like this, which feels project-based instead of user-based.

Reply:
I was thinking the same thing going through this code.

Although I feel like there needs to be a separation between user preferences (local to developer) and team preferences (saved via source control)